test/testSetup.ts

```diff
@@ -24,6 +24,7 @@ export class TestSetup {
   public projectFolderPath: string | undefined;
   private prompt: QuickOpenBox | InputBox | undefined;
   public scratchOrgAliasName: string | undefined;
+  public scratchOrgId: string | undefined;
```

Comment:
How does this get set?

Reply:
Why do we need this when we already know `scratchOrgAliasName`?

Reply:
`scratchOrgAliasName` is not present in `ScratchOrgInfo`, so we need the ID to identify it